src/ripple/resource/impl/Logic.h

```diff
@@ -421,13 +416,24 @@ class Logic
         return Disposition::ok;
     }
 
-    void acquire (Entry& entry, SharedState::Access& state)
+    void erase (Table::iterator iter)
     {
```

Comment:
Technically we do not: we already have the lock in our caller (see `periodicActivity` in the same file) but maybe it's a good idea to add it anyways,